src/saltext/vmware/modules/vmc_sddc.py

```diff
@@ -225,14 +225,35 @@ def create(
 
     deployment_type: String
         (Optional) Denotes if request is for a SingleAZ or a MultiAZ SDDC. Default is SingleAZ.
-        Possible values are: SingleAZ , MultiAZ
+        Possible values are: SingleAZ, MultiAZ
 
     host_instance_type: String
-        (Optional) The instance type for the esx hosts in the primary cluster of the SDDC.
+        (Optional) The instance type for the ESX hosts in the primary cluster of the SDDC.
         Possible values are: i3.metal, r5.metal, i3en.metal
 
-    msft_license_config : MsftLicensingConfig
+    msft_license_config :
         (Optional) Indicates the desired licensing support, if any, of Microsoft software.
+        It can be specified in the below format
+
+        .. code::
+
+            "msft_license_config": [
+                        {
+                            "academic_license": false,
+                            "mssql_licensing": "DISABLED",
+                            "windows_licensing": "DISABLED"
+
+                        }
+                    ]
+
+             where,
+            'academic_license' - Flag to identify if it is Academic Standard or Commercial Standard License.
+            'mssql_licensing' - The status MSSQL licensing for this SDDC’s clusters.
+                                Possible values are: DISABLED, CUSTOMER_SUPPLIED, ENABLED
+            'windows_licensing' - The status of Windows licensing for this SDDC’s clusters. Can be enabled, disabled, or customer’s.
```

Comment:
The `Possible values` below seems to make `Can be` redundant.

Reply:
Need to mention separately , as we are talking about two different parameters here